app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -115,8 +116,8 @@ class PrescriptionRepositoryAndroidTest {
 
     val softDeletedPrescription = database.prescriptionDao().getOne(prescription.uuid)!!
 
-    assertThat(softDeletedPrescription.updatedAt).isGreaterThan(prescription.updatedAt)
-    assertThat(softDeletedPrescription.createdAt).isEqualTo(prescription.createdAt)
+    assertThat(softDeletedPrescription.timestamps.updatedAt).isGreaterThan(prescription.timestamps.updatedAt)
```

Comment:
Let's use computed properties to get the `createdAt`, `updatedAt` & `deletedAt` so that existing references usages doesn't have to changed.

Reply:
I don't understand how the existing references wouldn't change after adding `createdAt`, `updatedAt` & `deletedAt` as computed properties ? Can you share code suggestion? Might help in understanding better

Reply:
We will be removing `createdAt`, `updatedAt` & `deletedAt` as constructor params and instead create a public property that will get those values from the `Timestamps`.

```kt
data class PrescribedDrug(
  val timestamps: Timestamps
) {
  val createdAt: Instant
    get() = timestamps.createdAt
}
```

doing this will let's continue using our older references and we only have to change the places where we need to update `Timestamps`.

Reply:
Removing createdAt, updatedAt & deletedAt as params is already done, but the idea of using computed properties for not affecting existing usages is great. Good approach. :)